metricbeat/module/system/process/process.go

```diff
@@ -32,17 +32,31 @@ type MetricSet struct {
 	cacheCmdLine bool
 }
 
+// includeTopConfig is the configuration for the "top N processes
+// filtering" feature
+type includeTopConfig struct {
```

Comment:
For some reason I thought the struct needed to be exported in order for go-ucfg to work on it via reflection. Researching further, I guess [since Go 1.6](https://golang.org/doc/go1.6#reflect) this isn't necessary.